atlasdb-workload-server/src/main/java/com/palantir/atlasdb/workload/store/AtlasDbTransactionStoreFactory.java

```diff
@@ -158,7 +158,15 @@ public static AtlasDbTransactionStoreFactory createFromConfig(
                 .build()
                 .serializable();
 
+        fastForwardTimestampToSupportTransactions3(transactionManager);
+
         return new AtlasDbTransactionStoreFactory(
                 transactionManager, installConfig.namespace().or(installConfig.keyValueService()::namespace));
     }
+
+    private static void fastForwardTimestampToSupportTransactions3(TransactionManager transactionManager) {
+        long currentTimestamp = transactionManager.getTimestampService().getFreshTimestamp();
+        long guaranteedTransaction3Timestamp = currentTimestamp + 10_000_000;
```

Comment:
yeah this will do. I was looking for the constant (it's `ADVANCEMENT_QUANTUM` in `KeyValueServiceCoordinationStore`), but doesn't look like it's exposed, and given this is test I think I'm ok with this